src/main/java/org/stellar/sdk/MemoWrongSizeException.java

```diff
@@ -0,0 +1,23 @@
+package org.stellar.sdk;
+
+/**
+ * Indicates that value passed to Memo has not the right size
+ * @see Memo
+ */
+public class MemoWrongSizeException extends RuntimeException {
+    private int size;
+    
+    public MemoWrongSizeException(int length) {
+        super();
+        this.size = length;
+    }
+
+    public MemoWrongSizeException(String message, int length) {
+        super(message);
+        this.size = length;
+    }
+    
+    public int getActualSize() {
+        return size;
```

Comment:
We should name all the variables containing the actual size a consistent name. It reduces the concepts the reader needs to digest.

```suggestion
    private int actualSize;
    
    public MemoWrongSizeException(int actualSize) {
        super();
        this.actualSize = actualSize;
    }

    public MemoWrongSizeException(String message, int actualSize) {
        super(message);
        this.actualSize = actualSize;
    }
    
    public int getActualSize() {
        return actualSize;
```